test/parallel/test-crypto-random.js

```diff
@@ -315,3 +315,180 @@ assert.throws(
   assert.strictEqual(desc.writable, true);
   assert.strictEqual(desc.enumerable, false);
 });
+
+
+{
+  const randomInts = [];
+  for (let i = 0; i < 100; i++) {
+    crypto.randomInt(1, 3, common.mustCall((err, n) => {
+      assert.ifError(err);
+      assert.ok(n >= 1);
+      assert.ok(n <= 3);
+      randomInts.push(n);
+      if (randomInts.length === 100) {
+        assert.ok(!randomInts.includes(0));
+        assert.ok(randomInts.includes(1));
+        assert.ok(randomInts.includes(2));
+        assert.ok(randomInts.includes(3));
```

Comment:
While it's most likely that after 100 tries we're going to have multiples of 1s, 2s, and 3s, there is still an improbable chance this test could fail. It would likely be enough to test that the value on each iteration is `1 <= n <= 3` and leave it at that.

Reply:
My math might be a bit off but I believe this should happen about once every ~406,561,180,000,000,000 test. If it fails, it's much more likely that a bug was introduced during a refactor (e.g. if the function starts returning integers the `1 <= n < 3` range, the simple `1 <= n <= 3` test would pass and the bug would go unnoticed).

Reply:
Then consider changing to a simple histogram type test. 

```js
const buckets = [0, 0, 0]
for (let i = 0; i < 100; i++) {
  const n = crypto.randomInt(1, 3)
  assert(n >= 1 && n <= 3)
  buckets[n-1]++
}
assert(buckets[0] > 0)
assert(buckets[1] > 0)
assert(buckets[2] > 0)
```

Should be much more efficient.

Reply:
And if the boundary conditions are what you're worried about, you can make it even more efficient by checking only that 1 and 3  appear at least once and ignore all between values.